packages/core/saved-objects/core-saved-objects-server/src/serialization.ts

```diff
@@ -11,6 +11,58 @@ import type {
   SavedObjectReference,
 } from '@kbn/core-saved-objects-common';
 
+/**
+ * A serializer that can be used to manually convert {@link SavedObjectsRawDoc | raw} or
+ * {@link SavedObjectSanitizedDoc | sanitized} documents to the other kind.
+ *
+ * @public
+ */
+export interface ISavedObjectsSerializer {
```

Comment:
Had to extract an explicit interface for the `SavedObjectsSerializer` implementation.